src/main/kotlin/controller/RacingGameController.kt

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import model.Car
+import service.RacingCarGameService
+import view.InputView
+import view.OutputView
+
+class RacingGameController {
+    private val inputView = InputView()
+    private val outputView = OutputView()
+
+    private val racingCarGameService = RacingCarGameService()
+    private var carsInfo = listOf<Car>()
+    private var tryCount = 0
+
+    fun run() {
+        getCarNames()
+        getTryCount()
+        playWholeRacing()
+        gerWinner()
+    }
+
+    fun getCarNames() {
+        try {
+            outputView.printCar()
+            val carsName = racingCarGameService.splitCarNames(inputView.inputName())
+            carsInfo = racingCarGameService.initCarInfo(carsName)
+        } catch (e: IllegalArgumentException) {
+            e.printStackTrace()
+            getCarNames()
+        }
+    }
+
+    fun getTryCount() {
+        try {
+            outputView.printTryCount()
+            tryCount = inputView.inputTryCount().toInt()
+        } catch (e: IllegalArgumentException) {
+            e.printStackTrace()
+            getTryCount()
+        }
+    }
+
+    fun playRound() {
+        carsInfo = racingCarGameService.moveCars(carsInfo)
+
+        outputView.printRoundResult(racingCarGameService.getRoundResult(carsInfo))
+    }
+
+    fun playWholeRacing() {
+        outputView.printRunResultMessage()
+        for (round in 1..tryCount) playRound()
```

Comment:
```suggestion
        repeat(tryCount - 1) {
            playRound()
        }
```
repeat 함수를 활용해보면 어떨까요?